runtimes/pythonrt/ak_runner/attrdict.py

```diff
@@ -0,0 +1,23 @@
+class AttrDict(dict):
```

Comment:
Yes. It's in https://github.com/autokitteh/autokitteh/blob/miki/events/runtimes/pythonrt/py-sdk/autokitteh.py which still at work. If it's urgent I'll make a new PR since I don't know how much time it'll take me to debug the event issues I have.